pkg/resiliency/endpoint/types.go

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package endpoint
+
+import (
+	"fmt"
+
+	"github.com/vishvananda/netlink"
+
+	"github.com/cilium/cilium/pkg/endpoint"
+)
+
+const (
+	qdiskClsact           = "clsact"
+	tcFilterParentIngress = 0xfffffff2
```

Comment:
What is this constant and where is it from? Can it change? Should't it live next to the code that sets up the tc filters so it doesn't accidentally get broken?

Reply:
Good catch! I piggy back off cmd/cleanup which defined this const. But it looks like netlink already export them. Axing...